model/Hardware/Classes/Taxonomy.md

```diff
@@ -0,0 +1,29 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Taxonomy
```

Comment:
Taxonomy doesn't quite make sense to me.   Are there some examples and use cases you've got documented to help illustrate this.   It's not clear how this class is contributing to ah heirarchical structure to me as well.